src/trace_reader_memtrace.cpp

```diff
@@ -76,7 +76,7 @@ void TraceReaderMemtrace::binaryGroupPathIs(const std::string &_path) {
                   error.c_str());
             return;
         }
-        module_mapper_ = module_mapper_t::create(directory_.modfile_bytes,
```

Comment:
hmm is this a current bug? If yes please extract it into a separate commit/pull request